src/bankAccount.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount) {
+  this.name = name;
+  this.amount = amount;
+  this.accountHistory = [`Initial: ${this.amount}`];
+};
+
+BankAccount.prototype = {
+  getInfo: function() {
+    return (`Name: ${this.name}, Amount: ${this.amount}$`);
+  },
+
+  addMoney: function(plusAmount, info) {
```

Comment:
не привязывайся к типу операции
```suggestion
  addMoney: function(amount, info) {
```
тоже самое withdraw